docs/authorization-configuration.md

```diff
@@ -36,7 +36,9 @@ auth: {
 - **scope** defines the permissions to request at login.
 - **redirect_uri** sets the URL to return to after login. The default application root is `/`.
 - **post_logout_redirect_uri** sets the URL to return after logout. The default URL is `/logout.html`.
-- **automaticSilentRenew** enables the renewal of the automatic token if it is supported by the server. The default value is `false`.
+- **automaticSilentRenew** enables the automatic silent renewal of the token if it is supported by the server. The default value is `false`. For this mechanism to work, browser must have third-party cookies support enabled.
```

Comment:
`For this mechanism to work, the browser must have third-party cookies support enabled.`

Reply:
corrected